modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/utils/RenderableMessage.java

```diff
@@ -59,18 +59,18 @@ private boolean extractPart(final Part part) throws MessagingException, IOExcept
 
         if (part.getContent() instanceof Multipart) {
 
-//            LOG.info("EXTRACT PART (" + n + ") " + "Multipart");
+//            LOG.log(Level.INFO, "EXTRACT PART ({0}) Multipart", n);
```

Comment:
Ugh.

I would either delete or uncomment those single-liners of logging. Maybe in a standalone change and everywhere so we can easily find it in history later when needed.

Reply:
Exactly

Reply:
Do you agree to still migrate them here and delete the migrated ones later in a different PR?

Reply:
yes, that's a good idea to have them separated.